### PR TITLE
Feature/getting started

### PR DESCRIPTION
**Pull request by jenwynne**
ganache and truffle up and running
readme updates done

### CODE REVIEW

File: contracts/YourTurnToken.sol [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+pragma solidity ^0.6.0;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+// Main YourTurn Token
+contract YourTurnToken is ERC20 {
+  constructor(uint256 initialBalance) ERC20("YourTurn", "YTT") public {
+    _mint(msg.sender, initialBalance);
+  }
+}
```
Comment (kinggs) [inline contracts/YourTurnToken.sol:11]: This is duplicated now. We need a single version of the file.

**Review by kinggs — CHANGES_REQUESTED** [review]
Thanks! Just fix the duplicated file, then we are good to go.

File: contracts/YourTurnToken.sol (hunk 1 above)
Comment (jenwynne) [reply contracts/YourTurnToken.sol]: Done

**Review by kinggs — APPROVED** [review]
All the truffle waffles!

**Pull request merged**